Automattic/woocommerce-payments
# Explicitly mention all devDependencies in the package.json file

Fixes #5817

#### Changes proposed in this Pull Request

We are using some packages without defining them in the `package.json` file. This PR adds the following eight packages to the file:

```
@jest/test-sequencer
@woocommerce/csv-export
@wordpress/dom-ready
@wordpress/url
iti
moment
react-transition-group
tinycolor2
```

For future reference, we still need to add the following five packages (these are showing errors and failed tests now) to the `package.json` file:

```
@woocommerce/data
@woocommerce/navigation
@wordpress/primitives
downshift
gridicons
```

#### Testing instructions

1. Clone the `fix/1616-define-dependenciesin-package-json-file` branch in your local machine.
2. Run `npm install` to install the newly added packages.
3. Run `npm run test:js` and make sure all the tests pass.
4. Run `npm run lint:js` and make sure there are no errors.



-------------------


**Post merge**
